lib/config.js

```diff
@@ -0,0 +1,10 @@
+/*!
+ * Copyright (c) 2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+
+config['stats'].storage.push({name: 'redis'});
```

Comment:
Is this the single place that these storage modules are added?  Looks like it would be easy to mistakenly add same module twice in two places.